### ISSUE TITLE
Translation of parshaEvent into Hebrew on Shabbats where there is no parsha

### ISSUE THREAD

**Issue by eliezerbloy** [hebcal/hebcal-es6#499]
שלום וברכה,
קודם כל תודה רבה על הספרייה המושקעת!!

אני רוצה להשתמש בספרייה כדי להציג את השם של השבת הנוכחית. בדרך כלל זה 

> "פרשת..."

ואני מגיע לזה כך:
```
let myHDate = new HDate(new Date(2025, 9, 11))
let sedra = getSedra(myHDate.getFullYear(), true).lookup(myHDate)
let parasha = sedra.parsha
const parshaEvent = new ParshaEvent(myHDate, parasha, true)
console.log(parshaEvent.render('he'))
```

 אבל בשבתות חול המועד וכדומה שאין פרשה זה אמור להיות 

> "שבת חול המועד פסח"

 וכדומה.
כרגע הוא מציג שבתות אלו בצורה פחות יפה להדפסה. למשל:
14/04/2025 הוא מציג 

> "פָּרָשַׁת פֶּסַח ז׳"

, 
11/10/2025 הוא מציג 

> "פרשת Sukkot Shabbat Chol ha-Moed"

תודה מראש

**Comment by mjradwin** [comment]
Thanks for the bug report. `ParshaEvent` is designed to work only with regular parashat haShavua on Shabbat, and not for holidays. 

Maybe you could use something like this instead?

```js
let myHDate = new HDate(new Date(2025, 9, 11))
let sedra = getSedra(myHDate.getFullYear(), true).lookup(myHDate)
let event
if (sedra.chag) {
  event = getHolidaysOnDate(myHDate, true)[0]
} else {
  let parasha = sedra.parsha
  event = new ParshaEvent(myHDate, parasha, true)
}
console.log(event.render('he')) // סוּכּוֹת ה׳ (חוה״מ)
```

**Comment by eliezerbloy** [comment]
תודה לך על המענה, זה מקדם אותי!
האם יש אפשרות להגיע לתוצאה כמו "שבת חוה"מ סוכות"?

**Comment by davidpodless** [comment]
אני גם עם אותה בעיה, מה שאני חושב לעשות זה לבנות json פשוט עם כל מקרי הקצה האלה.
key = desc, value = התרגום הרצוי לעברית
 ואם הערך נמצא שם - פשוט להחזיר את הערך שאנחנו רוצים

לא אידיאלי, אבל יפתור את הבעיה

**Comment by mjradwin** [comment]
There is some logic in the @hebcal/leyning package to render a title like "Shabbat Chol haMoed Sukkot" in both English and Hebrew. That API builds on top of @hebcal/core and also includes details on which exact verses are read on each day

https://hebcal.github.io/api/leyning/functions/getLeyningOnDate.html

**Comment by modeitsch** [comment]
PR #728 was merged and adds documentation for this case.

It clarifies that `ParshaEvent` is intended for regular weekly parsha readings, and points users to `getHolidaysOnDate()` or `@hebcal/leyning` / `getLeyningOnDate()` for Shabbatot with holiday readings such as Shabbat Chol HaMoed.

If this guidance is enough for the original question, I think this issue can be closed.

**Comment by eliezerbloy** [comment]
תודה!
אני לא בטוח שהבנתי, 
יש כרגע אפשרות לעבד כותרת כמו "שבת חוה"מ סוכות"?
זה נראה לי פורמט מקובל בלוחות שנה יהודיים.
זה חורג ממטרת הספרייה?

**Comment by modeitsch** [comment]
@eliezerbloy 
Yes, there is a way to get a title in that style, but not through `ParshaEvent`. The PR only documents this more explicitly: `ParshaEvent` is intended for regular weekly parsha readings. When Shabbat has a holiday Torah reading, such as Shabbat Chol HaMoed, the better API is `@hebcal/leyning`.

For example:

```
import {HDate} from '@hebcal/hdate';
import {getLeyningOnDate} from '@hebcal/leyning';

const hdate = new HDate(new Date(2025, 9, 11));
const reading = getLeyningOnDate(hdate, true, false, 'he');

console.log(reading.name.he);
```

**Comment by modeitsch** [comment]
I think PR #728 addressed the `ParshaEvent` confusion: `ParshaEvent` is intended for regular weekly parsha readings, and holiday Shabbat readings should use `getHolidaysOnDate()` or `@hebcal/leyning` instead.

That said, I think there may still be a separate display-format question here. `@hebcal/leyning` can provide holiday/Shabbat leyning titles for these dates, but the desired wording seems to be the common calendar-style Hebrew title, for example `שבת חוה״מ סוכות`.

So I would consider the documentation part handled, but maybe keep this open if the remaining desired feature is a dedicated formatted title for Shabbat Chol HaMoed / holiday Shabbat names in Hebrew.
